### PR TITLE
chore: bump libsession-util and use oxen fork

### PR DESCRIPTION
**Pull request by Bilb**

### CODE REVIEW

File: src/base_config.hpp [hunk 1]
```diff
@@ -15,8 +15,7 @@ namespace session::nodeapi {
 
 class ConfigBaseImpl;
 template <typename T>
-inline constexpr bool is_derived_napi_wrapper =
-        std::is_base_of_v<ConfigBaseImpl, T> && std::is_base_of_v<Napi::ObjectWrap<T>, T>;
+inline constexpr bool is_derived_napi_wrapper = std::is_base_of_v<Napi::ObjectWrap<T>, T>;
```
Comment (yougotwill) [inline src/base_config.hpp:18]: Noice

File: src/blinding/blinding.hpp [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+            if (obj.IsEmpty())
+                throw std::invalid_argument("blindVersionPubkey received empty");
+
+            assertIsUInt8Array(obj.Get("ed25519SecretKey"));
+            auto ed25519_secret_key =
+                    toCppBuffer(obj.Get("ed25519SecretKey"), "blindVersionPubkey.ed25519SecretKey");
+
+            auto keypair = session::blind_version_key_pair(ed25519_secret_key);
+            session::uc32 crappy_array_pk = std::get<0>(keypair);
```
Comment (yougotwill) [inline src/blinding/blinding.hpp]: rename crappy please

**Review by yougotwill — CHANGES_REQUESTED** [review]

File: src/blinding/blinding.hpp (hunk 2 above)
Comment (Bilb) [reply src/blinding/blinding.hpp]: renamed

**Review by yougotwill — APPROVED** [review]
LGTM 🚢

**Pull request merged**